presto-hive/src/main/java/com/facebook/presto/hive/statistics/MetastoreHiveStatisticsProvider.java

```diff
@@ -114,6 +114,7 @@ public TableStatistics getTableStatistics(ConnectorSession session, ConnectorTab
             }
             else {
                 rangeStatistics.setDistinctValuesCount(calculateDistinctValuesCount(partitionStatistics, columnName));
```

Comment:
what about adding `rangeStatistics.setDataSize(...)` for partition keys?

Reply:
We compute column stats for the partitioning keys -- since we know all the keys, we can do that.